models/cart.model.js

```diff
@@ -0,0 +1,16 @@
+const mongoose = require('mongoose');
+const { Schema } = mongoose;
+
+const cartSchema = new Schema({
+  _id: Schema.Types.ObjectId,
+  products: [
+    {
+      _id: { type: Schema.Types.ObjectId, ref: 'Product' },
+      qty: Number,
```

Comment:
hey pranshu, you can change the `qty` variable to a more verbose name for better readability